kayttooikeus-domain/src/main/java/fi/vm/sade/kayttooikeus/report/AccessRightReportRow.java

```diff
@@ -0,0 +1,51 @@
+package fi.vm.sade.kayttooikeus.report;
+
+import lombok.Generated;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.math.BigInteger;
+import java.util.Date;
+
+@Generated
+@Getter
+public class AccessRightReportRow {
+
+    private final BigInteger id;
+    private final String personName;
+    private final String personOid;
+    private final String organisationOid;
+    private final String accessRightName;
+    private final BigInteger accessRightId;
+    private final Date startDate;
+    private final Date endDate;
+    private final Date modified;
+    private final String modifiedBy;
+    @Setter
+    private String organisationName;
+
+    public AccessRightReportRow(
```

Comment:
lombok.AllArgsConstructor?

Reply:
tried, but for some reason didn't work with SqlResultSetMapping.ConstructorResult thus reverted to manually written constructor